src/api/window/window.ts

```diff
@@ -476,14 +476,14 @@ export class _Window extends Base {
     }
 
     /**
-     * Stops any current navigation the window is performing.
+     * Navigates the window to a specified URL.
+     * @param {string} url - The URL to navigate the window to.
      * @return {Promise.<void>}
-     * @tutorial Window.stopNavigation
+     * @tutorial Window.navigate
      */
-     public stopNavigation(): Promise<void> {
```

Comment:
@andreGarvin was it your intention to delete stopNavigate.

Reply:
Yes

Reply:
wait, why ?